### PR TITLE
deployment test

### PR DESCRIPTION
**Pull request by jinyangyang222**

### CODE REVIEW

File: modules/deployment/src/test/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformDeploymentTest.java [hunk 1]
```diff
@@ -10,46 +10,84 @@
...
         OclLoader oclLoader = new OclLoader();
         Ocl ocl = oclLoader.getOcl(new URL("file:./target/test-classes/ocl_test.yaml"));
 
-        CreateRequest deployRequest = new CreateRequest();
-        deployRequest.setName(ocl.getName());
-        deployRequest.setCsp(ocl.getCloudServiceProvider().getName());
-        deployRequest.setVersion(ocl.getVersion());
-        deployRequest.setFlavor(ocl.getFlavors().get(0).getName());
+        CreateRequest createRequest = new CreateRequest();
+        createRequest.setName(ocl.getName());
```
Comment (baixinsui) [inline modules/deployment/src/test/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformDeploymentTest.java:41]: When there are  more than one test methods need the same input data, we can use @BeforeClass to init the data Object

File: modules/deployment/src/test/resources/ocl_test.yaml [hunk 2]
```diff
@@ -78,32 +78,47 @@ deployment:
...
+        }
+    }
+    resource "huaweicloud_evs_volume" "myvol" {
+        name              = "volume"
+        availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
+        volume_type       = "SAS"
+        size              = 10
+    }
+    resource "huaweicloud_compute_volume_attach" "attached" {
+        instance_id = huaweicloud_compute_instance.basic.id
```
Comment (baixinsui) [inline modules/deployment/src/test/resources/ocl_test.yaml]: In order to save resources and time, We can simplify  the deployer in the ocl_test.yaml

File: modules/deployment/src/test/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformDeploymentTest.java (hunk 1 above)
Comment (jinyangyang222) [reply modules/deployment/src/test/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformDeploymentTest.java:37]: ok, i can do this

File: modules/deployment/src/test/resources/ocl_test.yaml (hunk 2 above)
Comment (jinyangyang222) [reply modules/deployment/src/test/resources/ocl_test.yaml:107]: I try to be as concise as possible

**Pull request merged**